swat/cas/table.py

```diff
@@ -3555,7 +3554,7 @@ def all(self, axis=None, bool_only=None, skipna=True, level=None, **kwargs):
 
     def any(self, axis=None, bool_only=None, skipna=True, level=None, **kwargs):
         '''
-        Return whether any elements in the column are True
+        Return True for each column with at least one True element
```

Comment:
Don't capitalize second "True".

Reply:
Return True for each column with at least one element treated as True

Reply:
I'm still not fond of the second literal 'True' here.  The values don't have to literally be 'True', they just have to be true by rules of truth (i.e., true == non-zero / non-empty-string / etc.).

Reply:
Okay. I changed the second 'True' to 'true' in any() and all().